Change config file for each handler separately (#18)

ZeroRin opened the pull request.

Dragon2fly commented on tests/test_edit_log_path.py:
```diff
@@ -0,0 +1,42 @@
+def test_edit_log_path():
+    config_file = Path("../logger_tt/log_config.json")
+    config = json.loads(config_file.read_text())
+    config['handlers']['urllib_file_handler'] = config['handlers']['error_file_handler']
```
Please add a comment line so that we would know immediately what this line does 6 months later.

Dragon2fly reviewed (changes requested):
Please exclude/remove `__pycache__` folder, change the annotation and add a little comment.

ZeroRin commented:
Actually what I did was that I removed the `__pycache__` directory which has been there 2 years ago, probably before you add `__pycache__` into `.gitignore`.

Added comments as requested

Dragon2fly reviewed (approved):
Thank you. I'll remove the __pycache__ folder later.
